test/cli/scenarios_to_cli/other /scenario_mutate_endpoint/policy.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion : kyverno.io/v1
+kind : ClusterPolicy
+metadata :
+  name : policy-endpoints
+spec :
+  rules:
+    - name: pEP
+      match:
+        resources:
+          kinds : 
```

Comment:
All policies need to have an `any` or `all` under the `match` and `exclude` blocks to be compatible with current language.